web/templates/user/index.html.eex

```diff
@@ -1,28 +1,58 @@
-<h2>Listing users</h2>
+<%= render Org.SharedView, "nav.html", assigns %>
 
-<table class="table">
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Email</th>
+<header>
+  <h2>
+    Listing
+    <%= if @current_user.role == "admin" do %>all<% end %>
+    <%= if @current_user.role == "member" do %>
+    members<% else %>users<% end %>
+  </h2>
+</header>
 
-      <th></th>
-    </tr>
-  </thead>
-  <tbody>
-<%= for user <- @users do %>
-    <tr>
-      <td><%= user.name %></td>
-      <td><%= user.email %></td>
-
-      <td class="text-right">
-        <%= link "Show", to: user_path(@conn, :show, user), class: "btn btn-default btn-xs" %>
-        <%= link "Edit", to: user_path(@conn, :edit, user), class: "btn btn-default btn-xs" %>
-        <%= link "Delete", to: user_path(@conn, :delete, user), method: :delete, data: [confirm: "Are you sure?"], class: "btn btn-danger btn-xs" %>
-      </td>
-    </tr>
-<% end %>
-  </tbody>
-</table>
-
-<%= link "New user", to: user_path(@conn, :new) %>
+<section class="card__list">
```

Comment:
Change this to. card-list. For BEM, the '__' means that this would be a necessary part of the .card class, but it is separate. if a .card-list can exist without a .card, then .card can stay the same and doesn't have to be .card-list__card. If it has to have one, then .card-list__card or card-list__item wold work